### PR TITLE
Create work branch;

### PR DESCRIPTION
**Pull request by OlegShackleford**
Did first home work 8-1\8-2

### CODE REVIEW

File: src/ht8_1/Phone.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    public void receiveCall (String name){
+        System.out.println("Звонит {"+ name +"}" + getNumber());
+    }
+
+    public void receiveCall(String name,String number)
+    {
+        System.out.println("Звонит " + name + "\n" +"Телефон " + number);
+    }
+
+    public void sendMessage (String[]...numbers){
```
Comment (dmitars) [inline src/ht8_1/Phone.java:33]: that's list of String arrays. Why don't use simple String array (or parameter with ...)?

File: src/ht8_2/BaseConverter.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package ht8_2;
+
+public class BaseConverter {
+    public void convert(double temp,Scale type){
```
Comment (dmitars) [inline src/ht8_2/BaseConverter.java:4]: Sense was to convert all temperatures between each other. K -> F,C; F -> K, C; C -> F,K

File: src/ht8_2/BaseConverter.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        } else if (type.equals(Scale.FAHRENHEIT))
+        {
+            double fahrenheit = temp * 1.8 + 32;
+            System.out.println("Температура: " + fahrenheit+"F");
+        } else if(type.equals(Scale.CELSIUS)){
+            System.out.println("Температура: " + temp+"C");
+        }
+    }
+
+    public enum Scale {
```
Comment (dmitars) [inline src/ht8_2/BaseConverter.java:19]: Better in different file

**Review by dmitars — APPROVED** [review]

**Comment by OlegShackleford** [comment]
Merge branches

**Pull request merged**